src/select/utils/DefaultProps.js

```diff
@@ -21,6 +22,10 @@ export const simpleSelectDefaults = {
   clearInputOnSelect: true,
   clearRenderer: null,
   focusPlaceholderComponent: FocusPlaceholder,
+  groupByKey: null,
+  groups: [],
+  groupTitle: null,
```

Comment:
Suggest `groupTitleKey` instead of `groupTitle`. Other then that, this looks good!